app/interactions/email_peer.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+class EmailPeer < ActiveInteraction::Base
+  string :peer_alias
+  string :message
+  object :contribution, class: Listing
+  object :user
+
+  def execute
+    peer_to_peer_email = PeerToPeerMatchMailer.peer_to_peer_email(
+      contribution,
+      peer_alias: peer_alias,
+      message: message,
+    )
+
+    status = Messenger.new(peer_to_peer_email, 'peer_to_peer_email').deliver_now
+
+    CommunicationLog.log_email(
+      email: peer_to_peer_email,
+      delivery_status: status,
+      person: contribution.person, # to person?
```

Comment:
@exbinary, Can you please confirm if the below assumption is right?

`CommunicationLog#person` - the person to whom the email was sent (whoever created the contribution)
`CommunicationLog#initiator` - the person who initiated the P2P flow

Reply:
Idk if the #person will *always* the person who created the Contribution. (Ideally later people will be able to reply to an initiator even if that person never created the "matching/opposite" Contribution. At this point I think we're automagically creating those matched items, so there still will be a "creator", tho idk if we're definitely setting "created_by" when we do that...)

We might want to change "person" to "recipient" and "initiator" to "sender" ?

Reply:
But, yeah, I wouldn't think the Person should be the current_user.person ???

Reply:
@maebeale I think we should rename `person` to `recipient` and `initiator` to `sender` as well (those are more semantic). 

When we initially implemented the P2P, we were setting both `person` & `initiator` attributes to [`current_user.person`](https://github.com/rubyforgood/mutual-aid/pull/730/files#diff-2d74dcaa2d57cb4ea0e3178afebb30be9726cb4995e0f158fca886ae7901c4c0R45). When I was refactoring the code, that struck as odd as we need to save both recipient & sender details but we were saving just the sender's details.

Reply:
yes, absolutely, @acherukuri ! i'm not sure how we ended up w where we were, but i def agree w your instincts. we will have both people/accounts at that point, right?

Reply:
Yes, we'll have both account details then.